app/Http/Controllers/ProductController.php

```diff
@@ -112,4 +113,15 @@ public static function cartItem(){
         $count = Cart::where('user_id', Auth::id())->count();
         return $count;
     }
+
+    public function cartList(){
+        $user_id = Auth::id();
+        $products = DB::table('cart')
```

Comment:
This code joins the "cart" and "products" table and finds products where product_id and cart.product_id are the same.